Src/IronPython/Compiler/Ast/CallExpression.cs

```diff
@@ -48,10 +50,14 @@ public class CallExpression : Expression, IInstructionProvider {
             get { return _target; }
         }
 
-        public IList<Arg> Args {
+        public Arg[] Args {
             get { return _args; }
         } 
 
+        public IList<Arg> ImplicitArgs {
```

Comment:
Maybe make this internal? Seems more like an implementation detail than something we want to expose in the ast.